kali_sg.tf

```
@@ -65,6 +51,30 @@ resource "aws_security_group_rule" "ingress_from_anywhere_to_kali_via_allowed_po
   to_port           = each.value["to_port"]
 }
 
+# Allow unfettered access between Kali and Teamserver instances
+resource "aws_security_group_rule" "kali_egress_to_teamserver_instances" {
```

Should we mention the ports in the resource name like we do elsewhere in this file?  I'm not tied to this suggestion, feel free to change it:
```suggestion
resource "aws_security_group_rule" "kali_egress_to_teamserver_instances_via_5000s" {
```

---

Please see commit 0a70a6e.